deployer/templates/hawkular-cassandra-node-dynamic-pv.yaml

```diff
@@ -114,6 +114,12 @@ objects:
             exec:
               command:
               - "/opt/apache-cassandra/bin/cassandra-docker-ready.sh"
+          lifecycle:
+            preStop:
+              exec:
+                command:
+                - "/opt/apache-cassandra/bin/cassandra-prestop.sh"
+          terminationGracePeriodSeconds: 1800
```

Comment:
I want to get away from having everything configurable as template options for the deployer.

If someone wants to configure this, they can still do so by editing the rc template.